packages/venia-ui/lib/components/Image/image.css

```diff
@@ -16,18 +16,13 @@
      */
 }
 
-.loaded {
+.image_positioned {
     display: block;
```

Comment:
So now we are just overlaying the image on top of the placeholder?

Reply:
Yes

Reply:
Outdated.